app/client/sass/tablet_view.scss

```diff
@@ -94,31 +132,40 @@
     margin: 0.25em;
     display: inline-block;
     float: right;
+    clear: both;
     font-size: 1.15em;
   }
 
   .job-information {
     section {
-      display: flex;
+      aside {
+        margin: 0em;
+        padding: 0em 1em;
+      }
     }
   }
 
   .thumbnail {
-    margin: 1.5em;
-    height: 200px;
-    width: 200px;
+    height: 240px;
```

Comment:
Same comment as before... specifying both dimensions could cause distortion in non-square images.